contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -19,6 +19,21 @@ import {IStakeRegistry} from "eigenlayer-middleware/interfaces/IStakeRegistry.so
 contract AlignedLayerServiceManager is ServiceManagerBase, BLSSignatureChecker {
     address aggregator;
 
+    // EVENTS
+    event NewTaskCreated(uint32 indexed taskIndex, Task task);
+
+    // STRUCTS
+    struct Task {
+        uint16 verificationSystemId;
+        bytes proof;
+        bytes pubInput;
+        uint32 taskCreatedBlock;
+    }
```

Comment:
Is the block needed ? Why is it there ? Also the fields are not correct but we can change it